monero-node/umbrel-app.yml

```diff
@@ -0,0 +1,22 @@
+manifestVersion: 1
+id: monero-node
+category: Finance
+name: Monero Node
+version: "0.18.2.2"
+tagline: A simple monero node runner
+description: >-
+  Monero node lets you transact over the secure Monero 
+  blockchain network without the need to trust other parties. 
+releaseNotes: >-
+  This is the most recent version of Monero node.
+developer: 
+website: https://getmonero.prg
+dependencies:
+repo: https://github.com/monero-project/monero
+support: https://github.com/monero-project/monero/issues
+port: 3800
+gallery:
+  - 1.jpg
```

Comment:
Were these images intended to be a part of this PR? I don't see them present in the existing files.

Reply:
yeah, these images are not included into the repository, they are shared in the PR. I also found it a strange design, but I believe it's due to github's limitations, not Umbrel's :)